recipes/openjdk.rb

```diff
@@ -42,7 +42,7 @@
   end
 end
 
-if node['platform'] == 'ubuntu'
```

Comment:
This change effectively disables the condition and removes the apt-repository configuration on ubuntu.